README.md

```diff
@@ -14,42 +14,82 @@ We use this prototype internally to LeanIX to generate SBOMs for our projects. I
 
 1. Run the Docker container from the latest image:
 
+GitHub:
 ```console
-docker run --pull=always \
+docker run --pull=always --rm \
            -v /var/run/docker.sock:/var/run/docker.sock \
            -v <temp-folder-to-be-used-for-storing-data>:/tempDir \
            -e MOUNTED_VOLUME='<temp-folder-to-be-used-for-storing-data>' \
+           -e LEANIX_HOST='<leanix-workspace-host>' \
+           -e LEANIX_REGION='<leanix-workspace-region>' \
            -e LEANIX_TOKEN='<leanix-technical-user-token>' \
+           -e GIT_PROVIDER='GITHUB' \
            -e GITHUB_TOKEN='<github-token>' \
            -e GITHUB_ORGANIZATION='<github-organization>' \
-           leanixacrpublic.azurecr.io/vsm-sbom-booster -rm
+           leanixacrpublic.azurecr.io/vsm-sbom-booster
 ```
+
+GitLab:
+```console
+docker run --pull=always --rm \
+           -v /var/run/docker.sock:/var/run/docker.sock \
+           -v <temp-folder-to-be-used-for-storing-data>:/tempDir \
+           -e MOUNTED_VOLUME='<temp-folder-to-be-used-for-storing-data>' \
+           -e LEANIX_HOST='<leanix-workspace-host>' \
+           -e LEANIX_REGION='<leanix-workspace-region>' \
+           -e LEANIX_TOKEN='<leanix-technical-user-token>' \
+           -e GIT_PROVIDER='GITLAB' \
+           -e GITLAB_TOKEN='<github-token>' \
+           -e GITLAB_GROUP='<github-organization>' \
+           leanixacrpublic.azurecr.io/vsm-sbom-booster
+```
+
 2. After a while your mapping inbox should be receiving new discovery items. These will need to be mapped by you once (see our [user documentation](https://docs-vsm.leanix.net/docs/discover-automate#create-your-service-baseline)).
 
 ### Environment Variables
 
+#### Runtime settings
 The first `-v` param is needed as the setup is a docker-in-docker setup and will need to mount the local docker runtime into the container. Under normal circumstances this param can be copied and pasted.
 
 The second `-v` param is the path to temporary folder that the `vsm-sbom-booster`will use to temporarily clone the projects to attempt to generate the SBOM. e.g. `~/output/temp`. Docker should have Read/Write access on this folder.
 
 `MOUNTED_VOLUME`: this is the same as the second `-v` param. It's required as the container needs an explicit env variable to do its job.
 
+`CONCURRENCY_FACTOR`(optional): the number of parallel jobs `vsm-sbom-booster` will use to generate SBOMs. Note: increasing this number will come at higher compute costs. Default: 3
+
+#### LeanIX configs
+
+`LEANIX_HOST`: The host name of your LeanIX workspace. e.g. For `https://acme.leanix.net` you would provide `acme`
+
+`LEANIX_REGION`: The region that your VSM workspace is hosted in. One of: eu|de|us|au|ca|ch
```

Comment:
@dsimpson-leanix i would object. we dont need both region and host. Example
1. the vsm workspace is located under `www.mycompany.leanix.net`: the MTM endpoint to fetch the bearer token is will be reached by calling `www.mycompany.leanix.net/services/mtm/v1/oauth2/token/`. This is quite common across the customer base i know. advantage is the customer doesnt need to know the region, which is a leanix internal concept.
2. calling the MTM API with the hosting region `www.eu.leanix.net/services/mtm/v1/oauth2/token/`: this call also reaches the API endpoint. 

I would prefer #1 as it's much more intuitive for any client. from the bearer token we can also then derive the hosting region.

Reply:
I don't think our documentation covers this clearly enough to not offer it as an option. However, we can merge the too pretty easily. I will make the HOST optional, and if not provided then it will use the region instead. Sound good to you?

Reply:
LGTM

Reply:
@dsimpson-leanix , would you think it would be feasible to have a single command for executing this regardless of the git provider? 

I was thinking that we might rename some of our env variables to more generic names (GIT_PROVIDER_HOST instead of two different parameters, GIT_PROVIDER_ORG representing organization or group ect) and switch to the correct implementation and default values based on the GIT_PROVIDER env variable.

@vg-leanix what do you think about this?

Reply:
@geoandri I tried to put myself in the customer's shoes when thinking about the command they would need to run. A customer will likely use EITHER GitHub OR GitLab OR BitBucket... If you had to set this up for your company, would you rather try to figure out how specific data from your git provider fits into the generic fields from this container, or would you rather have a set of fields that are clearly defined for your git provider and use the terms you already know? While a single run command makes our lives slightly easier developing this, it doesn't help the user experience. 

Happy to still make it generic if you say so @vg-leanix

Reply:
given that I would "only" forsee 4 Git Providers supported here, i would side with @dsimpson-leanix here to have env variables & a sample docker run command for every git provider.

Reply:
I am fine with one example per git provider and more descriptive env variables. When it comes to customer's perspective, I trust you guys 😉 .